src/layouts/Base.astro

```diff
@@ -154,7 +154,7 @@ const stars = fetch('https://api.github.com/repos/Eludris/elusite')
       }
 
       #content {
-        min-height: calc(100vh - 6em);
+        min-height: calc(100vh - 2.95rem - 86px);
```

Comment:
can you explain the significance of these figures? please follow the model used to explain the significance of E = mC² + AI